src/java.desktop/unix/classes/sun/awt/X11/XCreateWindowParams.java

```diff
@@ -81,11 +81,11 @@ public XCreateWindowParams delete(Object key) {
         return this;
     }
     public String toString() {
-        StringBuffer buf = new StringBuffer();
+        StringBuilder buf = new StringBuilder();
         Iterator<Map.Entry<Object, Object>> eIter = entrySet().iterator();
         while (eIter.hasNext()) {
             Map.Entry<Object, Object> entry = eIter.next();
-            buf.append(entry.getKey() + ": " + entry.getValue() + "\n");
+            buf.append(entry.getKey()).append(": ").append(entry.getValue()).append("\n");
```

Comment:
Would it be clearer if each append was on its own line?
```suggestion
            buf.append(entry.getKey())
               .append(": ")
               .append(entry.getValue())
               .append("\n");
```

Reply:
done